packages/npm-lit/demo/index.ts

```diff
@@ -0,0 +1,69 @@
+import { atom, Atom, peek } from '@reatom/core'
+import { withReatomElement, watch } from '../src/index.js'
+import { LitElement, html } from 'lit'
+
+const timer = atom(0)
+const countAtom = atom(0)
+
+const increment = atom(() => {
+  countAtom(countAtom() + 1)
+})
```

Comment:
`increment` should be `action` :) Or here is the better way:

```suggestion
const timer = atom(0, 'timer')
const count = atom(0, 'count').mix((target) => {
  increment: () => target((state) => state + 1)
})
```

Reply:
done